### PR TITLE
Fixed error in centroidal dynamics

### PR DESCRIPTION
**Pull request by echandler5956f**

### CODE REVIEW

File: examples/src/go1_test.cpp [hunk 1]
```diff
@@ -56,44 +52,44 @@ int main(int argc, char **argv)
...
-
-    GNUPlotInterface plotter(new_sol, cons);
-    plotter.PlotSolution({std::make_tuple(solver_interface.states()->q_index, solver_interface.states()->q_index + 3), std::make_tuple(solver_interface.states()->q_index + 3, solver_interface.states()->q_index + solver_interface.states()->nqb)},
-                         wrench_indices,
-                         {"Positions", "Orientations"},
-                         {{"x", "y", "z"}, {"qx", "qy", "qz", "qw"}},
-                         ee_plot_names,
-                         wrench_legend_names);
-    plotter.PlotConstraints();
+    // // TODO: Add interface to get the solution
```
Comment (Akshay5312) [inline examples/src/go1_test.cpp:55]: would it be prudent to just remove this at this point? It should be a separate helper function anyway.

File: galileo/legged-model/include/galileo/legged-model/LeggedBody.h
Comment (Akshay5312) [inline galileo/legged-model/include/galileo/legged-model/LeggedBody.h:1]: glad we finally made these changes!

**Review by Akshay5312 — APPROVED** [review]
looks good!

We should probably consider creating a separate helper, even possibly specific to the example if needed, to get and plot solutions

**Comment by Akshay5312** [comment]
Closes #58

File: examples/src/go1_test.cpp (hunk 1 above)
Comment (echandler5956f) [reply examples/src/go1_test.cpp:55]: Yep! Thats exactly what I was planning to do next.

**Pull request merged**